units/patch_intent.go

```diff
@@ -302,8 +302,16 @@ func (j *patchIntentProcessor) finishPatch(ctx context.Context, patchDoc *patch.
 	}
 
 	if j.intent.ReusePreviousPatchDefinition() {
-		patchDoc.VariantsTasks, err = j.getPreviousPatchDefinition(project, false)
+		err = j.setToPreviousPatchDefinition(patchDoc, project)
 		if err != nil {
+			grip.Error(message.WrapError(err, message.Fields{
+				"message":     "setting patch definition to the previous patch",
```

Comment:
Do we return this error after finish patch anyway, since we add it to the job? Not sure we need to double log.

Reply:
Do we also need to modify 320? (only failed?)

Reply:
So we probably should. However, I'm hesitant to do it right now because if it doesn't work well, I won't know, because we don't have any instances of it to monitor. We also haven't seen any errors associated with it so I don't think it's bothering anyone right now.

Reply:
removed logging